src/lib.rs

```diff
@@ -1,130 +1,47 @@
 use std::error::Error;
-use std::io;
+// config
+pub mod config;
+pub use crate::config::Config;
 
 // CLI input output
-
-fn take_user_input(prompt: &str) -> String {
```

Comment:
[nit]: Now that we removed these functions, is this comment here outdated?

Reply:
probably, but for now they give insights in what is contained in those modules. but guess it would be better if the module names would be more self explanatory then.

What do you think of renaming `cli.rs` to `cli_io.rs` to support removing these comments?

Reply:
👍🏻 I think that is a nice name indeed